common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -268,7 +253,8 @@ nav.sequence-nav {
       box-shadow: 0 1px 0 rgba(255, 255, 255, .7) inset;
       z-index: 1;
 
-      &.prev, &.next {
+      &.prev,
```

Comment:
If we're going to do this type of small NIT work, I'd rather us do it a clean up-based PR that does it across the application. Mind saving this change for that and reverting?